### DISCUSSION TITLE
Load Cockpit into a IFrame

### DISCUSSION THREAD

**Discussion by FTeinz** [cockpit-project/cockpit#19244] [Q&A] [answered]
I try to load Cockpi in a IFrame in Organizr. But getting the website has rejected the connection. Is there some way to allow it?

**Comment by martinpitt** [comment, marked as answer]
Indeed this will not work if the iframes come from a different origin, see https://cockpit-project.org/guide/latest/embedding.html . The way to make this work is to have a reverse proxy to make cockpit "appear" under the same origin as Organizr. See e.g. https://github.com/cockpit-project/cockpit/wiki/Proxying-Cockpit-over-Apache-with-LetsEncrypt or https://github.com/cockpit-project/cockpit/wiki/Proxying-Cockpit-over-NGINX for recipes.

**Comment by Ryderjj89** [comment]
Looking at this because I'm trying to get Cockpit to show up in Organizr and those docs don't mention anything (using nginx) on how to "make cockpit appear under the same origin".

Can the docs be updated with this information on how exactly to do that?

Edit: This fixed it for me with nginx proxy manager: proxy_hide_header X-Frame-Options;
